### PR TITLE
Navigation Basis

### PR DESCRIPTION
**Pull request by ZoroarkZark**
For this PR, I was creating the basis for navigation, and making a demoable version of it for use. This is meant to be a framework for creating other navigation on screens involved with the ActivitiesStack, and will likley change with Matthew's own implementations mixing with my own,

### CODE REVIEW

File: components/EventsList.tsx [hunk 1]
```diff
@@ -48,14 +51,20 @@ const EventsList = () => {
...
             data={events}
             renderItem={({ item }: ListRenderItemInfo<CurrentUserEvent>) => (
-              <View
-                style={{
-                  marginHorizontal: MARGIN_HORIZONTAL,
-                  marginVertical: MARGIN_VERTICAL
-                }}
+              <TouchableOpacity
+                onPress={() =>
+                  navigation.navigate("Event Details", { event: item })
```
Comment (ZoroarkZark) [inline components/EventsList.tsx:56]: Don't bake in navigation, separate navigation logic out into another component

File: stacks/ActivitiesStack.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import { EventDetailsProps } from "@screens/EventDetails/EventDetails"
+import { EventDetailsScreenNavWrapper } from "@screens/EventDetailsScreenNavWrapper"
+import { EventFormScreenProps } from "@screens/EventFormScreen"
+import { EventFormScreenNavWrapper } from "@screens/EventFormScreenNavWrapper"
+import {
+  LocationSearchPicker,
+  LocationSearchPickerProps
+} from "@screens/LocationSearch"
+
+const Stack = createStackNavigator()
```
Comment (ZoroarkZark) [inline stacks/ActivitiesStack.tsx:15]: Look into more navigation stack styles, might be better ones

File: app.json
Comment (seanim0920) [inline app.json:1]: Covered in app.config.ts

**Comment by seanim0920** [comment]
* Fix tests
* Play around with integrating Matthew's navigation PR
* Pass id's instead of objects in navigate()

**Review by seanim0920 — APPROVED** [review]

**Review by nicoletteL — APPROVED** [review]

**Pull request merged**